Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
nitpick: I usually add this entry below the members field (or in this case) below all other fields.

Reply:
Yes, good point - should they be alphabetical?

_I have probably made the same mistake on previous operators. I'll fix each as I spot them._

Reply:
I have raised an arch meeting discussion about TOML formatting / rules.

This instance is resolved by 2c299f7..a08108a